squad/api/rest.py

```diff
@@ -818,11 +818,28 @@ class TestRunSerializer(serializers.HyperlinkedModelSerializer):
     log_file = serializers.HyperlinkedIdentityField(view_name='testrun-log-file')
     tests = HyperlinkedTestsIdentityField(view_name='testrun-tests')
     metrics = HyperlinkedMetricsIdentityField(view_name='testrun-metrics')
+    status_summary = serializers.SerializerMethodField(read_only=True)
 
     class Meta:
         model = TestRun
         fields = '__all__'
 
+    def get_status_summary(self, obj):
```

Comment:
IMHO this is a bad idea. The totals should be calculated for the model and the API should only expose them. With the proposed implementation, calculation will happen with each API call. Since TestRun object doesn't change after it's created, there is no need to do that.